Library/Formula/fpc.rb

```diff
@@ -25,6 +25,9 @@ def install
     system "make", "install", "PP=#{fpc_compiler}", "PREFIX=#{prefix}"
 
     bin.install_symlink lib/"#{name}/#{version}/ppcx64"
+
+    # Prevent non-executable audit warning
+    rm_f Dir[bin/"*.rsj"]
```

Comment:
To my understanding, they are only useful for localization later, but not required for execution. There were only two files for `makeskel` and `ptop`.